chart/kubeapps/templates/kubeappsapis/config.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if gt (len .Values.clusters) 0 }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kubeapps.kubeappsapis-config.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: kubeappsapis
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data:
+  clusters.conf: |-
```

Comment:
Exactly, I just went this way because of the current structure of the repo, but I certainly don't like having two identical manifests. The point here is: will kubeops be entirely replaced by kubeapps-apis? if so, we can bear with it for the moment. If not, perhaps a good option is just creating a `shared` folder with common manifests like this one.

Reply:
Great, let's go with the `shared` folder. However, in this PR, I'd opt not to modify kubeops, just creating the `cm` in the shared folder. In a follow-up PR I'll do the change in kubeops (to avoid more uncertainty/instability here)